package.json

```diff
@@ -12,8 +12,14 @@
     "clean": "lerna run clean --stream",
     "build": "lerna run build --stream",
     "start": "http-server ./examples",
-    "test": "jest --runInBand --no-cache --config ./jest.config.js",
+    "coverage": "yarn coverage:jest; yarn coverage:karma",
+    "coverage:merge": "istanbul-merge --out coverage-merged.json coverage/coverage-final.json coverage/json/coverage.json",
+    "coverage:report": "yarn istanbul --include report coverage-merged.json --dir coverage-merged html",
+    "coverage:jest": "yarn jest --coverage",
+    "coverage:karma": "yarn karma --coverage",
```

Comment:
When I see a group of npm scripts that work together and maybe don't need to work independently, it may be time to have one single script file and call it from here instead? Maybe not a shell script, but a node.js script?

Reply:
I've always wanted to start using https://www.npmjs.com/package/execa for node.js scripts

Reply:
But I think we may want to run them independently. If a developer is working exclusively on near-membrane-dom, they will have no need for running `yarn coverage:jest`.

Reply:
Fair, as long as we have use cases 👍🏼